server.js

```diff
@@ -14,10 +17,72 @@ const welcomeMessage = {
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+let messages = [welcomeMessage];
 
 app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT);
+app.get("/messages", (req, res) => {
+  res.json(messages);
+});
+
+app.post("/messages", (req, res) => {
+  if(req.body.from.length == 0){
+    res.status(400).json("Please enter a from property")
+    return
+  }
+  if(req.body.text.length == 0){
+    res.status(400).json("please enter a text property")
+    return
+  }
+  let message = {
```

Comment:
This value doesn't change, so you can declare is as const instead.

The answers on this SO question give a good overview of why this is a good thing to do : https://stackoverflow.com/questions/136880/sell-me-on-const-correctness